recipes/cson/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "cson" %}
+{% set version = "0.7" %}
+{% set sha256 = "c923819ea207e10fd7e58de2920fa621af04be67c307fc2efa36402b253e33a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - speg
```

Comment:
There will be a race condition b/c `speg` is added in this PR together with `cson`. You can just re-start the CIs later though.